src/overlays/actors/ovl_En_Tanron5/z_en_tanron5.h

```diff
@@ -5,24 +5,61 @@
 
 struct EnTanron5;
 
-#define ENTANRON5_GET(thisx) ((thisx)->params)
+#define EN_TANRON5_GET_TYPE(thisx) ((thisx)->params)
 
-#define ENTANRON5_0 0
-#define ENTANRON5_100 100
-#define ENTANRON5_107 107
-#define ENTANRON5_108 108
-#define ENTANRON5_110 110
+typedef enum {
+    /*   0 */ EN_TANRON5_TYPE_STATIC,
+    /*   1 */ EN_TANRON5_TYPE_MAJORA_PILLAR_1,
+    /*   2 */ EN_TANRON5_TYPE_MAJORA_PILLAR_2,
+    /*   3 */ EN_TANRON5_TYPE_MAJORA_PILLAR_3,
+    /*   4 */ EN_TANRON5_TYPE_MAJORA_PILLAR_4,
+    /*   5 */ EN_TANRON5_TYPE_MAJORA_PILLAR_5,
+    /*   6 */ EN_TANRON5_TYPE_MAJORA_PILLAR_6,
+    /*   7 */ EN_TANRON5_TYPE_MAJORA_PILLAR_7,
+    /*   8 */ EN_TANRON5_TYPE_MAJORA_PILLAR_8,
+    /*   9 */ EN_TANRON5_TYPE_MAJORA_PILLAR_9,
+    /*  10 */ EN_TANRON5_TYPE_MAJORA_PILLAR_10,
+    /*  11 */ EN_TANRON5_TYPE_MAJORA_PILLAR_11,
+    /*  12 */ EN_TANRON5_TYPE_MAJORA_PILLAR_12,
+    /*  13 */ EN_TANRON5_TYPE_PYRAMID_RUIN_1,
+    /*  14 */ EN_TANRON5_TYPE_PYRAMID_RUIN_2,
+    /*  15 */ EN_TANRON5_TYPE_PYRAMID_RUIN_3,
+    /*  16 */ EN_TANRON5_TYPE_PYRAMID_RUIN_4,
+    /*  17 */ EN_TANRON5_TYPE_PYRAMID_RUIN_5,
+    /*  18 */ EN_TANRON5_TYPE_PYRAMID_RUIN_6,
+    /*  19 */ EN_TANRON5_TYPE_PYRAMID_RUIN_7,
+    /*  20 */ EN_TANRON5_TYPE_PYRAMID_RUIN_8,
+    /* 100 */ EN_TANRON5_TYPE_LARGE_RUIN_FRAGMENT_1 = 100,
+    /* 101 */ EN_TANRON5_TYPE_LARGE_RUIN_FRAGMENT_2,
+    /* 102 */ EN_TANRON5_TYPE_LARGE_RUIN_FRAGMENT_3,
+    /* 103 */ EN_TANRON5_TYPE_LARGE_RUIN_FRAGMENT_4,
+    /* 104 */ EN_TANRON5_TYPE_LARGE_RUIN_FRAGMENT_5,
+    /* 105 */ EN_TANRON5_TYPE_LARGE_RUIN_FRAGMENT_6,
+    /* 106 */ EN_TANRON5_TYPE_LARGE_RUIN_FRAGMENT_7,
+    /* 107 */ EN_TANRON5_TYPE_SMALL_RUIN_FRAGMENT_1,
+    /* 108 */ EN_TANRON5_TYPE_SMALL_RUIN_FRAGMENT_2,
+    /* 109 */ EN_TANRON5_TYPE_SMALL_RUIN_FRAGMENT_3,
+    /* 110 */ EN_TANRON5_TYPE_ITEM_DROP_1,
+    /* 111 */ EN_TANRON5_TYPE_ITEM_DROP_2
+} EnTanron5Type;
 
 typedef struct EnTanron5 {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ s16 unk_144;
-    /* 0x148 */ Gfx* unk_148;
+    /* 0x144 */ s16 timer;
+    /* 0x148 */ Gfx* dList;
     /* 0x14C */ ColliderCylinder collider;
-    /* 0x198 */ s16 unk_198;
-    /* 0x19A */ s16 unk_19A;
-    /* 0x19C */ f32 unk_19C;
-    /* 0x1A0 */ u8 unk_1A0;
-    /* 0x1A1 */ u8 unk_1A1;
+    /* 0x198 */ union {
+                    s16 ruinFragmentRotationalVelocityX;
```

Comment:
I was wondering about the wording, but looking in the current codebase, `rotationalVelocity` is used in multiple places, and `angularVelocity` is never used. And looking at code like this:
```c
        this->actor.shape.rot.x += this->ruinFragmentRotationalVelocityX;
        this->actor.shape.rot.y += this->ruinFragmentRotationalVelocityY;
```
Since it's changing a rotation value, it sort of makes sense to call it `rotationalVelocity`.

Reply:
`angularVelocity` is the same as `rotationalVelocity`, though `angularVelocity` is usually the term used in Physics. Personally since we are already using `rotationalVeclocity` lets use that and possibly change it in the future (I would also argue shortening to vel as well when the time comes)

Reply:
Yeah, that sounds like a plan to me. I can even make the switch to `angularVelocity` as soon as this PR is merged; I'll just do it all at once.